src/tally/tests/test_KDEKernel.cpp

```diff
@@ -24,9 +24,9 @@ class MockEpanechnikovKernel : public KDEKernel {
   }
 
   // not implemented
-  std::string get_kernel_name() const {}
-  int get_order() const {}
-  int get_min_quadrature(unsigned int i) const {}
+  std::string get_kernel_name() const { return std::string(); }
```

Comment:
Woah, definitely good catches by PullRequest here. Should we make an issue to figure out what these should be returning?

Reply:
These are required instantiations of pure virtual parent class methods, but are never tested.  Other child classes of `KDEKernel` are used to test these methods in their own implementations.  Not sure it makes sense to make up a response just so that we can test it.

Reply:
Ah, yeah I just missed that these are present to complete the class for testing purposes.